src/grid.js

```diff
@@ -8,5 +8,12 @@ import {
   color
 } from 'styled-system'
 
-export const Flex = styled(_Flex)([], fontSize, color)
```

Comment:
Instead of setting the `displayName` here, I think it would be better to submit a similar PR to [grid-styled](https://github.com/jxnblk/grid-styled) :smile:

Reply:
Seems like there is already one https://github.com/jxnblk/grid-styled/pull/32.

Reply:
@gregwym It may just be a matter of updating the version of `grid-styled` that Rebass uses then.